src/test/unit/utils/MovingAverageTest.cpp

```diff
@@ -26,7 +26,9 @@
 
 #include "nupic/types/Types.hpp"
 #include "nupic/utils/MovingAverage.hpp"
+#include "nupic/utils/SlidingWindow.hpp"
 
+using namespace std;
```

Comment:
Consider amending http://nupic.docs.numenta.org/prerelease/contributing/cpp-style-guide.html to reflect this sentiment.